include/aspect/material_model/rheology/strain_dependent.h

```diff
@@ -178,6 +206,15 @@ namespace aspect
            */
           std::vector<double> viscous_strain_weakening_factors;
 
+          /**
+           * The healing time scale used in the temperature dependent strain healing model.
```

Comment:
Reading further down, this variable has units of 1/s. Is this correct?

Reply:
Checking back in on this. Since the units are 1/s, isn't the value more akin to a "rate" rather than a "timescale"?